__DEFINES/game.dm

```diff
@@ -24,3 +24,11 @@
 #define MINIGUN_SLOWDOWN_WIELDED 8
 
 #define FIREAXE_SLOWDOWN 1.4
+
+#define COMMAND_POSITIONS list("Captain", "Head of Personnel", "Head of Security", "Chief Engineer", "Research Director", "Chief Medical Officer")
+#define ENGINEERING_POSITIONS list("Chief Engineer", "Station  Engineer", "Atmospheric Technician", "Mechanic")
+#define MEDICAL_POSITIONS list("Chief Medical Officer", "Medical Doctor", "Geneticist", "Virologist", "Paramedic", "Chemist")
+#define SCIENCE_POSITIONS list("Research Director", "Scientist", "Geneticist", "Roboticist", "Mechanic")
+#define CIVILIAN_POSITIONS list("Head of Personnel", "Bartender", "Botanist", "Chef", "Janitor", "Librarian", "Internal Affairs Agent", "Chaplain", "Clown", "Mime", "Assistant")
+#define CARGO_POSITION list("Head of Personnel", "Quartermaster", "Cargo Technician", "Shaft Miner")
```

Comment:
the other macros have POSITION**S**, these two have POSITION
also don't we have a file for job defines?

Reply:
That files is lower down the file tree than the bloodcult files.

Reply:
*shrug